quimb/evo.py

```diff
@@ -542,8 +586,9 @@ def update_to(self, t):
             with continuous_progbar(self.t, t) as pbar:
                 if self._int_step_callback is not None:
                     def solout(t, y):
-                        self._int_step_callback(t, y, self._ham)
+                        int_stop = self._int_step_callback(t, y, self._ham)
                         pbar.cupdate(t)
+                        return int_stop
```

Comment:
Minor, but since ``int_stop`` as a function is used elsewhere, clearer to rename this ``int_stop_res`` or something (for those of us who hastily ctrl+f replace etc).

Reply:
Agreed. Done.